libs/designer/src/lib/ui/templates/styles.less

```diff
@@ -59,4 +59,14 @@
         width: 200px;
         margin-left: 6px;
     }
+}
+
+.msla-template-connection-name {
+    display: flex;
+    flex-direction: row;
```

Comment:
NIT: flex direction is by default in `display: flex` there is no need to add it.